static/js/util/util.js

```diff
@@ -221,3 +223,23 @@ export function makeProfileImageUrl(profile) {
 export function getPreferredName(profile) {
   return profile.preferred_name || SETTINGS.name || SETTINGS.username;
 }
+
+export function calculateDegreeInclusions(profile) {
+  let found = false;
+  let inclusions = {};
+  for (const { value } of EDUCATION_LEVELS) {
+    inclusions[value] = !found;
+    if (value === profile.edx_level_of_education) {
+      // every level after this point is higher than the user has attained according to edx
+      found = true;
+    }
+  }
```

Comment:
ok so here we're looping through `'HIGH_SCHOOL'`, `'ASSOCIATE'`, etc and setting `inclusions[value] = true`, until we encounter a `value` which is equal to the level of education from edx - then we set `found = true`, and we'll set the rest of the `values` from `EDUCATION_LEVELS` to false.

I get what this code does but it took me 5 minutes to get there - here's a suggestion for how to refactor it so it reads more clearly:

```js
let highestLevelFound = false;
let inclusions = {};
for (const { value } of EDUCATION_LEVELS) {
  inclusions[value] = highestLevelFound ? false : true;
  if (value === profile.edx_level_of_education) {
    // every level after this point is higher than the user has attained according to edx
    highestLevelFound = true;
  }
}
```

again, just a suggestion about how to make it clearer, use your judgement about what's best.